recipes/configs/llama2_eleuther_eval.yaml

```diff
@@ -0,0 +1,30 @@
+# Config for EleutherEvalRecipe in eleuther_eval.py
+#
+# To launch, run the following command from root torchtune directory:
+#    tune eleuther_eval --config llama2_eleuther_eval
+
+# Model Arguments
+model:
+  _component_: torchtune.models.llama2.llama2_7b
+
+checkpointer:
+  _component_: torchtune.utils.FullModelTorchTuneCheckpointer
+  checkpoint_dir: /tmp/llama/
+  checkpoint_files: [finetuned_model.pt]
```

Comment:
Where is this coming from? Might be nice to align with our output checkpoint file format so that it works out of the box

Reply:
It depends on the epoch, so not sure what the nice solution is here.